etc/try_parser.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+# Copyright 2023 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import json
+from enum import Enum
+import sys
+
+
+class Layout(str, Enum):
+    none = 'none'
+    layout2013 = '2013'
+    layout2020 = '2020'
+    all = 'all'
```

Comment:
This could be a [`Flag`](https://docs.python.org/3/library/enum.html#enum.Flag) type to avoid the ambiguity?